app/Http/Controllers/ChapterController.php

```diff
@@ -331,18 +331,15 @@ public function edit(Chapter $chapter)
      *
      * @return $this|bool|\Illuminate\Http\RedirectResponse
      */
-    public function update(Chapter $chapter)
+    public function update(Chapter $chapter, Request $request)
     {
         if (($redirect = $this->checkPermissions('EDIT_SHIP')) !== true) {
             return $redirect;
         }
 
-        $validator =
-            Validator::make($data = Request::all(), Chapter::$updateRules);
+        $this->validate($request, Chapter::$updateRules);
```

Comment:
This is straight out of the Laravel docs, and is the new best practice for validation